code/datums/elements/strippable.dm

```diff
@@ -178,6 +182,9 @@
 /// Performs an alternative action on this strippable_item.
 /// `has_alternate_action` needs to be TRUE.
 /datum/strippable_item/proc/alternate_action(atom/source, mob/user)
+	SHOULD_CALL_PARENT(TRUE)
+	if(SEND_SIGNAL(user, COMSIG_CAN_STRIP, source) & COMPONENT_CANT_STRIP)
```

Comment:
This doesn't make sense to be on the same signal (even when you do make it `TRY_STRIP`).

Signals, generally, should come from *one* place. Furthermore, this doesn't even have the same signature, since it doesn't provide the item. Make a separate signal that someone is trying to use an alternate action.

Reply:
I don't know what I was smoking